recipes-aos/aos-communicationmanager/files/aos_communicationmanager.cfg

```diff
@@ -15,10 +15,12 @@
     },
     "SMController": {
         "CMServerURL": ":8093",
+        "FileServerURL": "aossm:8094",
```

Comment:
We should listen on all interface I guess. `:8094`

Reply:
done